data/src/main/scala/com/exini/dicom/data/DicomParts.scala

```diff
@@ -100,9 +106,9 @@ object DicomParts {
       val headerBytes =
         if (explicitVR)
           if (vr.headerLength == 8)
-            tagToBytes(tag, bigEndian) ++ ByteString(vr.toString) ++ shortToBytes(length.toShort, bigEndian)
+            tagToBytes(tag, bigEndian) ++ CharacterSets.encode(vr.toString) ++ shortToBytes(length.toShort, bigEndian)
```

Comment:
The header part is never encoded as anything other than ISO-IR 6 so CharacterSets.encode is not necessary. Shouldn't brake anything though so it's no biggie.

Reply:
Right, it is encoded using utf8 here which should be the same as ISO_IR 6 for the characters in a VR name